NavigationReactNative/src/android/app/src/main/java/com/navigation/reactnative/IconResolver.java

```diff
@@ -0,0 +1,143 @@
+package com.navigation.reactnative;
+
+import android.content.Context;
+import android.graphics.drawable.Animatable;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.view.MenuItem;
+
+import com.facebook.drawee.backends.pipeline.Fresco;
+import com.facebook.drawee.controller.BaseControllerListener;
+import com.facebook.drawee.interfaces.DraweeController;
+import com.facebook.drawee.view.DraweeHolder;
+import com.facebook.imagepipeline.image.ImageInfo;
+import com.facebook.imagepipeline.image.QualityInfo;
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.uimanager.PixelUtil;
+import com.facebook.react.views.toolbar.DrawableWithIntrinsicSize;
```

Comment:
We need to copy over DrawableWithIntrinsicSize because [it's been removed from React Native 0.61](https://github.com/facebook/react-native/commit/cbbbb455dda313cca1a1763cc5453079ce87bc73)

We need to upgrade the zoom sample to the latest React Native to make sure that it works. I upgrade by deleting the zoom sample and recreating it but you need to be careful if you want to do it this way because you might lose the deep linking functionality. If you want to delete and recreate then let me know and I'll write out the steps required.

Reply:
File added.
I am not updating the zoom sample to the latest React Native because after doing `npm install`, lots of lines in `package-lock.json` changed and I don't know if I am doing right. Maybe because we use different npm version?

Reply:
Nevertheless, I tested it out and the sample is working on React Native v0.61 now.

Reply:
No problem, just let me know how you tested it on React Native v0.61

Reply:
I've created a [PR that updates the zoom sample to v0.61](https://github.com/emerson233/navigation/pull/1). Please merge it into this PR if you're happy

Reply:
> No problem, just let me know how you tested it on React Native v0.61

Just changed `package.json` like this.
```
"react": "16.9.0",
"react-native": "0.61.0"
```
And doing a `npm install`.